tools/docker/mu-plugins/01-monorepo.php

```diff
@@ -0,0 +1,154 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Plugin Name: Monorepo Helper
+ * Description: A common place for monorepo things.
+ * Version: 1.0
+ * Author: Automattic
+ * Author URI: https://automattic.com/
+ * Text Domain: jetpack
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Jetpack\Docker\MuPlugin;
+
+/**
+ * Monorepo Tools.
+ */
+class Monorepo {
+	/**
+	 * Path to monorepo.
+	 *
+	 * @var string
+	 */
+	protected $monorepo;
+	/**
+	 * Path to monorepo plugins.
+	 *
+	 * @var string
+	 */
+	protected $plugins;
+	/**
+	 * Path to monorepo packages.
+	 *
+	 * @var string
+	 */
+	protected $packages;
+
+	/**
+	 * Class constructor.
+	 */
+	public function __construct() {
+		$this->monorepo = '/usr/local/src/jetpack-monorepo/';
+		$this->plugins  = $this->monorepo . 'projects/plugins/';
+		$this->packages = $this->monorepo . 'projects/packages/';
+	}
+
+	/**
+	 * Property Getter
+	 *
+	 * @param string $var Property to get.
+	 *
+	 * @throws Exception If the requested property does not exist.
+	 */
+	public function get( $var ) {
+		if ( is_string( $var ) && isset( $this->$var ) ) {
+			return $this->$var;
+		}
+		throw new Exception( "Class property $var does not exist." );
+	}
+
+	/**
+	 * The same as Core's get_plugins, without forcing a passed value to be within the wp-content/plugins folder.
+	 *
+	 * @param string $plugin_folder Folder to find plugins within.
+	 */
+	private function get_plugins( $plugin_folder = '' ) {
+		$cache_plugins = wp_cache_get( 'monorepo_plugins', 'monorepo_plugins' ); // Updated cache values to not conflict.
+		if ( ! $cache_plugins ) {
+			$cache_plugins = array();
+		}
+
+		if ( isset( $cache_plugins[ $plugin_folder ] ) ) {
+			return $cache_plugins[ $plugin_folder ];
+		}
+
+			$wp_plugins  = array();
+			$plugin_root = WP_PLUGIN_DIR;
```

Comment:
```suggestion
		$wp_plugins  = array();
		$plugin_root = WP_PLUGIN_DIR;
```

Reply:
It was a straight copy/paste from core, so I have no preference about the spacing.